src/vivarium_public_health/dataset_manager/artifact.py

```diff
@@ -31,16 +31,22 @@ def __init__(self, path: str, filter_terms: List[str]=None):
             A set of terms suitable for usage with the ``where`` kwarg for ``pd.read_hdf``
         """
 
+        Artifact.create_hdf_with_keyspace(path)
```

Comment:
This can (and should be) a `self.create_hdf_with_keyspace(path)`.  Pretty much always.  Even for class methods.  It doesn't matter here at all, but that's how you ensure inheritance works properly.